cli/resource.go

```diff
@@ -72,3 +77,33 @@ func runResourceAdd(args *docopt.Args, client *controller.Client) error {
 
 	return nil
 }
+
+func runResourceRemove(args *docopt.Args, client *controller.Client) error {
+	provider := args.String["<provider>"]
+	resource := args.String["<resource>"]
+
+	res, err := client.GetResource(provider, resource)
+	if err != nil {
+		return err
+	}
+
+	err = client.DeleteResource(provider, resource)
```

Comment:
Would it make sense for `DeleteResource` to return the resource which was deleted, thus avoiding the `GetResource` above?